mava/configs/default/mat.yaml

```diff
@@ -0,0 +1,11 @@
+defaults:
+  - logger: logger
+  - arch: anakin
+  - system: mat/mat
+  - network: transformer
+  - env: rware
```

Comment:
```suggestion
  - env: rware  # [cleaner, connector, gigastep, lbf, mabrax, matrax, rware, smax]
```

Reply:
Done. I didn't include cleaner and connector since MAT doesn't have a CNN and these envs don't have vector obs in Mava yet.